packages/apps/dev-wallet/src/pages/create-profile/create-profile.tsx

```diff
@@ -54,19 +79,61 @@ export function CreateProfile() {
   }
   return (
     <>
-      <Box margin="md">
-        <Heading variant="h5">Create wallet</Heading>
-        <Text>Enter a password to encrypt the wallet data with that</Text>
+      <Box className={authCard}>
```

Comment:
I think it's good to separate this card together with a back button to a component as the same card is used also on other pages so it can be reused